static/app/components/group/tagFacets/tagFacetsDistributionMeter.tsx

```diff
@@ -16,14 +17,15 @@ import useOrganization from 'sentry/utils/useOrganization';
 
 import {TagFacetsStyles} from '.';
 
-const COLORS = ['#402A65', '#694D99', '#9A81C4', '#BBA6DF', '#EAE2F8'];
+const COLORS = ['#3A3387', '#5F40A3', '#8C4FBD', '#B961D3', '#FEEBF9'];
 
 type Props = {
   project: Project;
   segments: TagSegment[];
   title: string;
   totalValues: number;
   colors?: string[];
+  expanded?: boolean;
```

Comment:
If we're adding a new prop, should we just name it something like `expandByDefault` so it doesn't look like it's a prop that you can pass down and control its open/close state?